corsheaders/__init__.py

```diff
@@ -1 +1,2 @@
 __version__ = '1.2.2'
+from .checks import check_settings
```

Comment:
It's slightly better to do this in an `AppConfig.ready` https://docs.djangoproject.com/en/1.10/ref/applications/ , but don't worry about that